package.json

```diff
@@ -112,6 +112,7 @@
     "cookie-session": "^2.0.0-alpha.1",
     "cookies-js": "^1.2.3",
     "cors": "^2.8.3",
+    "dd-trace": "^0.5.3",
```

Comment:
I noted this [here](https://docs.datadoghq.com/tracing/setup/nodejs/#requirements):

> ...it benefits significantly from the performance improvements introduced in Node >=8.3

I see the `8.x.x` spec in `package.json` and the `node:8` in `Dockerfile`, but can't tell _precisely_ what version Force uses currently.

Reply:
Then it'll like be on 8.11 which is the latest of the 8 branches (assuming we re-create the docker images on deploys etc)

Reply:
That we do...

```
$ hokusai staging run 'node --version'
v8.11.3
```

@orta can you pin the version in the Dockerfile to the tag `node:8.11.3`?

Reply:
Yep, done 👍